stdlib/sys.mli

```diff
@@ -32,6 +32,16 @@ val executable_name : string
     on the platform and whether the program was compiled to bytecode or a native
     executable. *)
 
+val runtime_executable : string
```

Comment:
Nit, but from a naming perspective, it would be better in my opinion to suffix this with `_name` to emphasize the similarity with `executable_name`.

Reply:
I find `Sys.executable_name` quite confusing: I'm never sure if it is a filename or file path. If you want something else here I'd rather suggest `Sys.runtime_executable_path` or `Sys.runtime_executable_filepath`.